components/dashboard/Dashboard.js

```diff
@@ -229,6 +243,29 @@ export default function Dashboard(props) {
               completedTasks={completedTasks}
               limit={historyLimit}
             />
+            <Box my={3}>
+              <Card>
```

Comment:
Thoughts on putting this (and its dialog hooks) in its own component?

Reply:
Not sure I understand -- do you mean putting the toggle buttons for showing & hiding the modal components into the ActivityInputDialog and UpcomingInterviewDialog?